package.json

```diff
@@ -29,7 +29,9 @@
   "dependencies": {
     "@apidevtools/swagger-parser": "^10.0.1",
     "@readme/emojis": "^2.1.0",
+    "@readme/ui": "^1.14.2",
     "babel-polyfill": "^6.26.0",
+    "oas": "^6.1.0",
```

Comment:
This isn't technically used in the preview site, but without it v5 of `oas` was getting loaded instead of v6 and it was breaking the entire explorer. 😰